### PR TITLE
Add keras and tensorflow integrations

### PR DESCRIPTION
**Pull request by hubertjaworskineptune**

### CODE REVIEW

File: neptune/experiment.py [hunk 1]
```diff
@@ -195,6 +195,10 @@ def send_artifact(self, artifact):
                           upload_tar_api_fun=self._client.extract_experiment_output,
                           experiment_id=self.internal_id)
 
+    def send_graph(self, graph_id, value):
+
```
Comment (dominik-jastrzebski) [inline neptune/experiment.py]: Add a docstring.

File: neptune/internal/integration/keras.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    _integrate_with_keras()
+
+    _integrated_with_keras = True
+
+
+def _integrate_with_keras():
+    try:
+        import keras
+    except ImportError:
+        raise ValueError('Requested integration with keras while keras is not installed.')
```
Comment (dominik-jastrzebski) [inline neptune/internal/integration/keras.py]: Define a `LibraryNotInstalled` exception in `exceptions.py` and throw it.

File: neptune/internal/integration/keras.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    def get_current_global_experiment():
+        pass  # TODO get current experiment from global state
```
Comment (dominik-jastrzebski) [inline neptune/internal/integration/keras.py]: A leftover?

File: neptune/internal/integration/tensorflow.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        return
+    _integrate_with_tensorflow()
+    _integrated_with_tensorflow = True
+
+
+def _integrate_with_tensorflow():
+    try:
+        import tensorflow
+    except ImportError:
+        raise ValueError('Requested integration with tensorflow while tensorflow is not installed.')
```
Comment (dominik-jastrzebski) [inline neptune/internal/integration/tensorflow.py]: Define a `LibraryNotInstalled` exception in `exceptions.py` and throw it.

File: neptune/internal/integration/tensorflow.py [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+    def get_current_global_experiment():
+        pass # TODO get current experiment from global state
```
Comment (dominik-jastrzebski) [inline neptune/internal/integration/tensorflow.py]: A leftover again?

File: neptune/internal/integration/keras.py (hunk 3 above)
Comment (hubertjaworskineptune) [reply neptune/internal/integration/keras.py]: After rebasing this will be replaced by a dynamic 'global' experiment getter

**Pull request merged**